content/en/tracing/advanced/runtime_metrics.md

```diff
@@ -26,7 +26,7 @@ Enable runtime metrics collection in the tracing client to gain additional insig
 {{< tabs >}}
 {{% tab "Java" %}}
 
-JVM metrics collection is enabled by default. It can be disabled with one configuration parameter in the tracing client, either through a system property, `-Ddd.jmxfetch.enabled=false`, or through an environment variable, `DD_JMXFETCH_ENABLED=false`.
+JVM metrics collection is enabled by default since Java tracer v0.29.0 . It can be disabled with one configuration parameter in the tracing client, either through a system property, `-Ddd.jmxfetch.enabled=false`, or through an environment variable, `DD_JMXFETCH_ENABLED=false`.
```

Comment:
```suggestion
JVM metrics collection is enabled by default for Java tracer v0.29.0+. It can be disabled with one configuration parameter in the tracing client, either through a system property, `-Ddd.jmxfetch.enabled=false`, or through an environment variable, `DD_JMXFETCH_ENABLED=false`.
```